lib/pfconfig/namespaces/resource/isolation_passthroughs.pm

```diff
@@ -0,0 +1,144 @@
+package pfconfig::namespaces::resource::isolation_passthroughs;
```

Comment:
Same idea as above, we need to remove duplicated logic.
Let me know if you want me to do it